lethal-squirrels/Bangazon-Site
# PICS

## REFERENCE

Ticket: none

## DESCRIPTION
pics on homepage, fixed size on the detail page, added paths to the initailizer

## STEPS TO TEST
1. Burn down your database
2. run `update-database`
3. run program and see the pics MUAHAHA
